libfwupdplugin/fu-udev-device.c

```diff
@@ -743,6 +743,28 @@ fu_udev_device_get_sysfs_path (FuUdevDevice *self)
 	return NULL;
 }
 
+/**
+ * fu_udev_device_get_number:
+ * @self: A #FuUdevDevice
+ *
+ * Gets the device number, if any.
+ *
+ * Returns: integer, or 0 if unavailable
```

Comment:
comment needs fixing, should be G_MAXUINT64

Reply:
Well, it's a a bit funcky but it's not untrue. In the case of a random device where numbering isn't available, the return value will be zero. When udev isn't available it'll be G_MAXUINT64. How about `Returns: integer, 0 if the number is unavailable, or %G_MAXUINT64 if udev is not available`?

Reply:
That's good